.github/workflows/merge_queue.yml

```
@@ -0,0 +1,869 @@
+
+permissions:
+  pull-requests: write
+  id-token: write
+  contents: write
+  statuses: write
+
+jobs:
+  context:
+    name: Context
+    if: ${{ needs.access.outputs.granted == 'true' }}
+    runs-on: ${{ fromJSON(inputs.runs-on) }}
```

actionlint (1.7.4)

property "runs-on" is not defined in object type {}

(expression)